en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/P/Protection_from_Evil_and_Good.md

```diff
@@ -1,5 +1,21 @@
 #### Protection from Evil and Good
-<!-- TODO Check and tag this spell -->
+<!-- markdownlint-disable link-image-reference-definitions -->
+[_metadata_:spell_name]:- "Protection from Evil and Good"
+[_metadata_:spell_level]:- "1"
+[_metadata_:spell_school]:- "abjuration"
+[_metadata_:ritual]:- "false"
+[_metadata_:casting_time_amount]:- "1"
+[_metadata_:casting_time_unit]:- "action"
+[_metadata_:range]:- "Touch"
+[_metadata_:target]:- "one willing creature"
+[_metadata_:components_verbal]:- "true"
+[_metadata_:components_somatic]:- "true"
+[_metadata_:components_material]:- "true"
+[_metadata_:components_material_description]:- "holy water or powdered silver and iron, which the spell consumes"
```

Comment:
Interestingly, while the material components are consumed the spell doesn't specify an amount or a value. So it could be just a tiny amount, making the consumption of the materials basically irrelevant.
This is true in the WotC SRD, the A5e SRD and in D&D Beyond; so I'm guessing it wasn't changed through errata. Fascinating.